documentation/src/main/asciidoc/topics/cache_api.adoc

```diff
@@ -1,8 +1,9 @@
 [[cache_api]]
-= The Cache API
+= Cache API
 
-== The Cache interface
-{brandname}'s Caches are manipulated through the link:{javadocroot}/org/infinispan/Cache.html[Cache] interface.
+== Cache interface
+{brandname} provides a link:{javadocroot}/org/infinispan/Cache.html[Cache]
+interface for creating, configuring, and managing Caches.
```

Comment:
well, technically, you can't create, configure or manage cache via Cache. That would be the CacheManager.

Reply:
true. good catch.